aikau/src/test/resources/alfresco/core/PageTest.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2005-2016 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This test assesses the CoreRwd mixin as applied to AlfMenuBarPopup
+ *
+ * @author Richard Smith
+ * @author Dave Draper
+ */
```

Comment:
It's a test page, so won't show up in the JSDoc, but it probably won't hurt to add one. Do we want to be doing this for all files now (purely for the benefit of the developer editing the file)?